### ISSUE TITLE
Showing Express Name 

### ISSUE THREAD

**Issue by m7mdcc** [shalva97/taobao-tracking-numbers#1]
Hi , Thanks for your effort in this script . 

I try to do  : 
```
 m.request({method: "GET", url:'https://buyertrade.taobao.com/trade/json/transit_step.do?bizOrderId=' + orderId ,headers: {
        "Content-Type": "application/json; charset=utf-8",
        "Accept": "application/json"
    }}).then(data => {
        appstate('Tracking num: ' + data.expressId + ' -- Company :'+  data.expressName )
    })
```

but its show garbage text , could you help here ? 

Another notes : 

* Its not working on pagination , so if you go to page 2 or 3 , its will not work 
* We need to add sleep between each request cause taobao flag our activity as unusual traffic :S We blocked ! 

Thanks

**Comment by shalva97** [comment]
yeah, I can reproduce it. Probably encoding issue, because page has charset=gbk in the html, asked on few Discord channels, hopefully someone will know the solution....

| Its not working on pagination
| We need to add sleep between each request cause taobao flag our activity as unusual traffic :S We blocked !
these are also bugging me. Its been few month I have not written JS and forgot how mithril works. so yeah.... cant promise any new features.
